games-strategy/augustus/files/augustus-3.0.1-uncapped.patch

```diff
@@ -0,0 +1,16 @@
+From 1d3a24d74bbea85198c2c5d739a16dcc139af18b Mon Sep 17 00:00:00 2001
+From: Horea Christian <chr@chymera.eu>
+Date: Sat, 26 Dec 2020 12:59:15 -0500
+Subject: [PATCH] Unlimited number of Grand Temples.
+
+--- a/src/building/monument.h	2021-05-17 09:54:41.000000000 -0400
++++ a/src/building/monument.h	17:16:52.327542581 -0400
```

Comment:
On a side-note, it's unimportant but you can remove the useless timestamp cruft after the .h

Devmanual has a handy sed to clean patches when need to do this often:
https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#clean-patch-howto